tools/doc/type-parser.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+const nodeDocUrl = '';
+const jsDocUrl = 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/' +
+  'Reference/Global_Objects/';
+const jsGlobalTypes = [
+  'Error', 'Number', 'Object', 'Boolean', 'String', 'Function', 'Symbol',
+  'Array', 'Uint8Array', 'Uint16Array', 'Uint32Array', 'Int8Array',
+  'Int16Array', 'Int32Array', 'Uint8ClampedArray', 'Float32Array',
+  'Float64Array', 'Date', 'RegExp', 'ArrayBuffer', 'DataView', 'Promise',
+  'Null'
```

Comment:
`String`, `Number` and `Boolean` should link to some page on the primitive, not the Object version. The Object versions of each have weird nuances we do not technically support oftentimes, and we definitely don't return those. :)